code/game/atoms_movable.dm

```diff
@@ -13,13 +13,15 @@
 	var/datum/thrownthing/throwing = null
 	var/throw_speed = 2 //How many tiles to move per ds when being thrown. Float values are fully supported
 	var/throw_range = 7
-	var/no_spin = 0
-	var/no_spin_thrown = 0
-	var/moved_recently = 0
+	var/no_spin = FALSE
+	var/no_spin_thrown = FALSE
+	var/moved_recently = FALSE
 	var/mob/pulledby = null
 	var/atom/movable/pulling
+	/// Face towards the atom while pulling it
+	var/face_while_pulling = FALSE
```

Comment:
I feel like a better/alternative method of doing this would be to utilize the weight class / move force stuff

https://github.com/ParadiseSS13/Paradise/pull/13350/files

Reply:
I've been looking into the code for that, and I'm not sure I can make it work properly.
The stuff with Mice and Drones not being able to drag things is all done in an override for `start_pulling()`, so I feel like it'd be a lot cleaner to just set a variable on the objects instead.

Of course I might be misreading it or missing something obvious, but the current way seems better overall.